projects/templates/projects/view_project.html

```diff
@@ -133,43 +133,92 @@ <h3>Project Users</h3>
 </form>
 {% endif %}
 
-<h4>Current Users</h4>
-<ul class="list-unstyled">
+<br>
+<br>
+<table width=100%>
+  <tr>
+    <th>Username</th>
+    <th>First Name</th>
+    <th>Last Name</th>
+    <th>Email</th>
+    <th>Role</th>
+    <th></th>
+  </tr>
   {% for u in users %}
-  <li>
-      {% if is_pi and u.username != request.user.username %}
-      <form role="form" method="post" style="display:inline">
-        {% csrf_token %}
-        <input type="hidden" name="user_ref" value="{{ u.username }}">
-        <button type="submit" class="btn btn-xs btn-link" name="del_user"><i class="fa fa-minus-square text-danger"></i><span class="sr-only">Remove user</span></button>
-      </form>
-    {% endif %}
-    <strong>{{ u.username }}</strong>{% if u.email %}: {{ u.first_name }} {{ u.last_name }} ({{ u.email }}){% endif %}
-  </li>
+  <tr>
+    <td>{{ u.username }}</td>
+    <td>{{ u.first_name }}</td>
+    <td>{{ u.last_name }}</td>
+    <td>{{ u.email }}</td>
+    <td>
+       <form role="form" method="post" style="display:inline">
+         {% csrf_token %}
+         <input type="hidden" name="user_ref" value="{{ u.username }}">
+         <select id="role_{{ u.email }}" name="user_role" class="form-select role_selector" disabled>
```

Comment:
I think the user ID is probably safer to put here than the email, from an encoding (and XSS injection) standpoint.